src/server/response.cpp

```diff
@@ -412,6 +412,12 @@ ContentResponse::ContentResponse(const std::string& root, bool verbose, const st
   m_mimeType(mimetype)
 {
   add_header(MHD_HTTP_HEADER_CONTENT_TYPE, m_mimeType);
+  if ( !startsWith(m_mimeType, "application/pdf") ) {
```

Comment:
@veloman-yunkan Have you tested that this actually catches PDF files? E.g., what if the mimeType isn't set? You might want to check also the file name with a case-insensitive regex like `/\.pdf$/i`.

Reply:
> Have you tested that this actually catches PDF files?

On the [test case](https://master.download.kiwix.org/zim/.hidden/dev/lilote_fr_test_2023-01.zim) that I am using - it does.